# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/sendmail/libresolv.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./devtools/OS/Linux~	Mon Dec 27 22:54:08 1999
+++ ./devtools/OS/Linux	Wed Mar  8 16:40:40 2000
@@ -1,7 +1,7 @@
 #	$Id: libresolv.patch,v 1.3 2003/01/10 12:12:51 clifford Exp $
 define(`confDEPEND_TYPE', `CC-M')
 define(`confMANROOT', `/usr/man/man')
-define(`confLIBS', `-ldl')
+define(`confLIBS', `-ldl -lresolv')
 define(`confEBINDIR', `/usr/sbin')
 APPENDDEF(`confLIBSEARCH', `crypt nsl')
 
